.buildkite/pipeline.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: 'load-test'
+    agents:
+      queue: 'default'
+      docker: '*'
+    timeout_in_minutes: 15
+    plugins:
+      - docker-compose#v3.0.0:
```

Comment:
`3.0.0`? is this strict versioning? Can we make it `<4` a.k.a `^3.0.0`? I'm asking due to lack of familiarity with buildkite

Reply:
It appears they must be specified exactly